docs/source/cloudlabtutorial/index.rst

```diff
@@ -0,0 +1,126 @@
+Cloudlab Tutorial 
+=====================================
+
+Getting Started
+-----------------
+
+- This tutorial assumes that you have access to the openNetVM `CloudLab <https://cloudlab.us/>`_ 
+    - To gain access, you must first create an account, choose to join a project and enter "GWCloudLab" to request access
+
+Setting up SSH Keys for CloudLab
+-----------------
+
+CloudLab allows you to upload an SSH key
+
+- Open a terminal and cd to the directory you want to store the keys in
+
+    Generally, on a Windows machine, all SSH-related files are found in following location: /c/Users/PC_USER_NAME/.ssh/
+    
+To generate an SSH Key:
+
+- Type :code:`ssh-keygen` to make an ssh-key
+
+- Give it a name and an optional password
+
+- The public key has the extension .pub
+
+
+To upload the key:
+
+- Go to cloudlab, and click on your username in the top-right corner
+
+- Go to "manage SSH keys"
+
+- Press "Load from file" and select the public key file
+
+- Once it's loaded in, select "add key"
+
+.. image:: ../images/cloudlab-ssh.png
+   :width: 600
+
+Starting a ONVM Profile With 1 Node on CloudLab
+-----------------
+
+- Click on experiments in the upper-left corner, then start an experiment
+
+- For "select a profile" you can choose the desired profile (onvm is the most updated/recommended)
+
+- For "parameterize" use 1 host, and for node type, you can keep it as is or select a different one. 
+
+    If you're running into trouble runnign the manager, selecting the c220g5 node may assist you
+    
+- For "finalize" you can just click next
+
+- For "schedule," you don't have to make a schedule, leaving it blank will start it immediately 
+
+Connecting to the Node via VSCode
+-----------------
+
+Before connecting, you must have uploaded your SSH key, and started an experiment
+You also must have these VSCode extensions:
+    Remote - SSH
+    Remote - SSH: Editing Configuration Files (may come preinstalled with Remote SSH)
+    Remote - Containers
+    Remote - WSL
+
+There are 2 easy ways to connect:
+
+1) Via a terminal
+
+- Open a VSCode terminal and cd inside your .ssh folder 
+
+- :code:`ssh -p 22 -i <privatekeyname> <user>@<address>`
+
+- Your <user>@<address> can be found by going to your experiment and clicking on "list view," it should be under "SSH Command"
+
+.. image:: ../images/ssh-connect.png
+   :width: 600
```

Comment:
I suggest you move this part up to the end of the prior section. Have it there just to show a quick way for users to test that their experiment started correctly.  Then have this section just be about VS Code remote dev

Reply:
Do you need the VSCode SSH extension to remotely connect via the command line? If that is the case, then it should go after the extensions section.

Reply:
no, you won't need the SSH extension to use SSH in the console (the extension just allows you to connect your entire VS Code window to a remote server.